app/v2/template/post_template.py

```diff
@@ -28,9 +29,11 @@ def post_template_preview(template_id):
 
     check_placeholders(template_object)
 
+    subject = template_object.subject if template.template_type != SMS_TYPE else None
+
     resp = create_post_template_preview_response(template=template,
-                                                 body=str(template_object),
-                                                 url_root=request.url_root)
+                                                 subject=subject,
```

Comment:
Small point.

It would be nicer if `create_post_template_preview_response` took in `template` and `template_object`. That way the logic for handling subject and returning `str(template_object)` can be abstracted into that function.